peers.json is not created

**Describe the bug**
The creation of peers.json is not robust. The file is not generated.

**To Reproduce**
Steps to reproduce the behavior:
1. compile the code 
2. run the simulation
3. Got error

[
  {
    "NetAddr": "127.0.0.1:12002",
    "PubKeyHex": "0x0481CDABD15A88A5EAF36A1B78FC203F3284232B793C2B0E6B8B20AD04975682FDB39183510C604EAF86CBFB9A67F4F766DCC23DFC91ACE36954598927D5F636A2"
  },
  {
    "NetAddr": "127.0.0.1:12003",
    "PubKeyHex": "0x04F02664BF95DA2C79BC4E634D18066E8552D39F26D3CC0111F045BFD802D5BBF655DCBE447DF5ECE63A5C2F1B3753E463DEBFE4A818539903723A7014D0C6299D"
  },
  {
    "NetAddr": "127.0.0.1:12004",
    "PubKeyHex": "0x04F08C3C9E25FD7FD65588C952A4CD721CE947B35F8DD11F4579FC64C62CAD8B056874589855DD93662DF7E906F705425F1024A9E3AFA9281AD82246D0A4A33F5B"
  }
]
cat: /home/qn/work/src/github.com/Fantom-foundation/go-lachesis/nodes/peers.json: No such file or directory

Steps:
Make a fresh clone: 
$git clone xxx
$make clean vendor proto build
$n=3 entry=main logshold=-1,-1,-1,85,0,0 BUILD_DIR="$PWD" ./scripts/multi.bash
----

Output and Error:
<========== start of output =====
$ n=3 entry=main logshold=-1,-1,-1,85,0,0 BUILD_DIR="$PWD" bash -x ./scripts/multi.bash
+ set -euo pipefail
+ declare -i OPTIND=1
+ declare -i debug=0
+ declare args=
+++ dirname ./scripts/multi.bash
++ cd ./scripts
++ pwd
+ declare -r DIR=/home/qn/work/src/github.com/Fantom-foundation/go-lachesis/scripts
+ getopts d opt
+ shift 0
+ '[' '' = -- ']'
+ '[' 0 == 1 ']'
+ /home/qn/work/src/github.com/Fantom-foundation/go-lachesis/scripts/multi_build.bash ''
# of logical CPUs:  4
# of physical CPUS: 2
# command-line-arguments
/tmp/go-link-318665185/000002.o: In function `mygetgrouplist':
/workdir/go/src/os/user/getgrouplist_unix.go:16: warning: Using 'getgrouplist' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-318665185/000001.o: In function `mygetgrgid_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:38: warning: Using 'getgrgid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-318665185/000001.o: In function `mygetgrnam_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:43: warning: Using 'getgrnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-318665185/000001.o: In function `mygetpwnam_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:33: warning: Using 'getpwnam_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-318665185/000001.o: In function `mygetpwuid_r':
/workdir/go/src/os/user/cgo_lookup_unix.go:28: warning: Using 'getpwuid_r' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
[
  {
    "NetAddr": "127.0.0.1:12002",
    "PubKeyHex": "0x044A869F291D672268D52F0F00A71D1DF6EBA757B2FB12D1C71E48A9E48E2951EB87374D3340FA14ED0CC5712E64AA358C80F43B2FBE9964EDF45E27B01697A513"
  },
  {
    "NetAddr": "127.0.0.1:12003",
    "PubKeyHex": "0x04EFC9C9288F7043684BEE8257F7879ABA5CFCE03C2DC3C966467AD64208B1D0FD7384D63AD398F8771CA2DCFF4613CF928422777ED9184CEE946B97E5BA67B1E7"
  },
  {
    "NetAddr": "127.0.0.1:12004",
    "PubKeyHex": "0x04EBE320F92C429E78B4964209F4B2B7C4B8B49C83AD97ECFCA311F064B288B35860CAAB91E0CBFF2EAF28A04754ECFB629E0C5F9BFEB9EA29D7366B89DA2FA7AD"
  }
]
cat: /home/qn/work/src/github.com/Fantom-foundation/go-lachesis/nodes/peers.json: No such file or directory
======= end of output ====>

--- a/README.md
+++ b/README.md
@@ -97,7 +97,7 @@ Create an 3 node lachesis cluster with:
   - [git](https://git-scm.com)
   - [Go](https://golang.org)
   - [Glide](https://glide.sh)
-  - [batch-ethkey](https://github.com/SamuelMarks/batch-ethkey) with: `go get -v github.com/SamuelMarks/batch-ethkey`
+  - [batch-ethkey](https://github.com/SamuelMarks/batch-ethkey) with: `go get -u github.com/SamuelMarks/batch-ethkey`
   - [protocol buffers 3](https://github.com/protocolbuffers/protobuf), with: installation of [a release]([here](https://github.com/protocolbuffers/protobuf/releases)) & `go get -u github.com/golang/protobuf/protoc-gen-go`
 
 ### Protobuffer 3

